coeus-db/coeus-db-sql/src/main/resources/org/kuali/coeus/coeus-sql/RELEASE-SCRIPTS/KC-RELEASE-6_0_0-SCRIPT/KC-RELEASE-6_0_0-Upgrade-MYSQL.sql

```diff
@@ -31,6 +31,8 @@
 \. ./../../current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-7889_B000.sql
 \. ./../../current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-7922_B000.sql
 \. ./../../current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-8027_B000.sql
+\. ./../../current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-8128_B000.sql
+\. ./../../current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-8159_B000.sql
 \. ./../../current_mysql/6.0.0/dml/KC_DML_01_KRACOEUS-8411_B000.sql
```

Comment:
this 8159 file is not longer in the repo and is not needed. This file will be generated manually so can you remove these two from the PR ?